src/Public/Add-CCMGroupMember.ps1

```diff
@@ -106,8 +111,18 @@ function Add-CCMGroupMember {
             Name        = $current.Name
             Id          = $current.Id
             Description = $current.Description
-            Groups      = @($GroupCollection)
-            Computers   = @($ComputerCollection)
+            Groups      = if ($GroupCollection.Count -gt 0) {
+                $GroupCollection.ToArray()
+            }
+            else {
+                $null
+            }
+            Computers   = if ($ComputerCollection.Count -gt 0) {
+                $ComputerCollection.ToArray()
+            }
+            else {
+                $null
+            }

```

Comment:
This construction still breaks, it looks like... the JSON conversion in WinPS strips off the array wrapper if there's only one element. Looks like the "proper" way to do this is just `Computers = $ComputerCollection` / `Groups = $GroupCollection`, it seems to work properly for empty arrays and all.